ui/appui-layout-react/src/appui-layout-react/state/internal/TabStateHelpers.ts

```diff
@@ -16,7 +16,7 @@ import { category } from "./NineZoneStateHelpers";
 /** @internal */
 export function createTabState(id: TabState["id"], args?: Partial<TabState>): TabState {
   return {
-    allowedPanelTargets: undefined,
+    allowedPanelTargets: args?.allowedPanelTargets,
```

Comment:
This can be removed, since it will be set when spreading the `args`?

Reply:
Done.